<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgColorSpaceSelector</class>
 <widget class="QWidget" name="WdgColorSpaceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>146</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblColorSpaces">
       <property name="text">
        <string>&amp;Model:</string>
       </property>
       <property name="buddy">
        <cstring>cmbColorModels</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisCmbIDList" name="cmbColorModels">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblColorModels">
       <property name="text">
        <string comment="Label of Color Depth in Color Space Selector dialogue">Depth:</string>
       </property>
       <property name="buddy">
        <cstring>cmbColorModels</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisCmbIDList" name="cmbColorDepth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblProfiles">
       <property name="text">
        <string>Profi&amp;le:</string>
       </property>
       <property name="buddy">
        <cstring>cmbProfile</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisSqueezedComboBox" name="cmbProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnInstallProfile">
       <property name="toolTip">
        <string>Install a new profile from a file</string>
       </property>
       <property name="whatsThis">
        <string>Install a new profile from a file.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="bnAdvanced">
     <property name="text">
      <string>Color Space Browser</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>KisSqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>KisSqueezedComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>